app/models/profile.rb

```diff
@@ -23,6 +23,7 @@ class Profile < ApplicationRecord
     gpo_verification_pending_NO_LONGER_USED: 3, # deprecated
     verification_cancelled: 4,
     in_person_verification_pending_NO_LONGER_USED: 5, # deprecated
+    expired_profile_under_fraud_review: 6
```

Comment:
We've been trying to avoid adding new values to `deactivation_reason`, instead favoring storing timestamps for state transitions (e.g. `fraud_review_pending_at`).

(I'm assuming that if the user doesn't go to the post office for 30 days, then their profile is truly expired and they need to restart proofing.)

If the user never visits the post office (regardless of fraud status), I think we should set an `in_person_verification_expired_at` timestamp on the profile (and clear `in_person_verification_pending_at`) to indicate the expiration happened--see `Profile.deactivate_due_to_gpo_expiration` for a template to use.

If the enrollment has expired and the user has also been flagged for fraud, it seems like there are a couple of options:

1. Set `fraud_rejection_at` timestamp to indicate that the profile was rejected due to fraud concerns
2. Set `fraud_review_pending_at` to the date/time the IPP enrollment was _created_ (i.e. 30 days ago), then let `FraudRejectionDailyJob` clean it up / set `fraud_review_rejection_at`.

From there you should be able to pull the affected profiles by searching for those with `in_person_verification_expired_at` and `fraud_rejection_at` set.

Reply:
i like the idea of using `deactivate_due_to_gpo_expiration` as a template for creating an ipp deactivation. Maybe something like `deactivate_due_to_ipp_expiration_during_fraud_review`  since we're not deactivating profiles currently for going past the expiration